part-spec/usbc.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/usbc.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "usbc",
+    "oneOf": [
+        {
+            "title": "usb-c pd controller",
+            "type": "object",
+            "required": [
+                "componentID",
+                "pdVersion"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "pdVersion": {
+                    "description": "version of power delivery spec implemented by controller",
+                    "examples": [
+                        "pd2",
+                        "pd3"
```

Comment:
Should the examples detail both the revision and version, given that the PD 3.1 spec introduced the Revision message which will report both?